src/components/HomePage.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import Header from "./Header";
+import Main from "./Main/Main";
+import Sidebar from "./Sidebar";
+
+interface HomeProps {
```

Comment:
[Suggestion] When an interface is not exported, a common practice is to name it just `Props`, as there's usually only one component per file.